code/processes/reporter.q

```diff
@@ -115,10 +115,13 @@ runreport:{[tab]
 		.timer.once[startts;fp;"Reporter - ",string tab`name];
 		`..timerids upsert 1!select id,periodend:nextrun from .timer.timer where fp~/:funcparam;
 	:()];
-	/ - if the current time is within the start and end timestamps, use current time as start time, else use the startts
+        / - if timezone is specified convert the start and end timestamps to the time used by the system 
+	startts:?[not null tab[`timezone];ltime[.tz.ttz[`GMT;tab[`timezone];startts]];startts];
+        endts:?[not null tab[`timezone];ltime[.tz.ttz[`GMT;tab[`timezone];endts]];endts];
```

Comment:
do definitely we want to use `ltime` here? I think that will depend on whether the process is running with `-localtime` flag set to true or false?